cmd/hope/utils/resources.go

```diff
@@ -66,8 +70,41 @@ func GetIdentifiableResources(names *[]string, tags *[]string) (*[]hope.Resource
 
 	// If any name wasn't found, error out.
 	if len(nameMap) != 0 {
-		return nil, fmt.Errorf("Failed to find resources with names: %s", strings.Join(*maputil.MapStringBoolKeys(&nameMap), ","))
+		return nil, fmt.Errorf("failed to find resources with names: %s", strings.Join(*maputil.MapStringBoolKeys(&nameMap), ","))
 	}
 
 	return &returnSlice, nil
 }
+
+// For each parameter from a file, load the file and populate the base64
+//   values of the files into the properties.
+// Does nothing to deduplicate keys.
+// All plain parameters will exist in the list before file parameters.
+func RenderParameters(directParameters, fileParameters []string) ([]string, error) {
```

Comment:
Feels like it should be `FlattenParameters`, not `RenderParameters`. Rendering seems like it would actually be applying to the base definition, rather than just loading files and returning a single list.